code/modules/food/kitchen/cooking_machines/container.dm

```diff
@@ -102,6 +105,8 @@
 
 /obj/item/weapon/reagent_containers/cooking_container/AltClick(var/mob/user)
 	do_empty(user)
+	food_items = 0

```

Comment:
Shouldn't this be set by `do_empty()`?

Reply:
This is the only way I've been able to figure out how to empty the container, other than sink tipping which gets rid of the reagents. I don't know if I can mess with do_empty() here safely.